android/jni/com/mapswithme/core/logging.cpp

```diff
@@ -64,18 +59,21 @@ void AndroidAssertMessage(SrcPoint const & src, string const & s)
 
 void InitSystemLog()
 {
-#ifdef MWM_LOG_TO_FILE
-  SetLogMessageFn(&LogMessageFile);
-#else
   SetLogMessageFn(&AndroidLogMessage);
-#endif
 }
 
 void InitAssertLog()
 {
   SetAssertFunction(&AndroidAssertMessage);
 }
 
+void ToggleDebugLogs(bool enabled)
+{
+  if (enabled)
+    g_LogLevel = LDEBUG;
+  else
+    g_LogLevel = LINFO;
```

Comment:
I left this level as it was. I think, it was done so logs in Crashlytics would be more useful.